src/mame/machine/tosh1000_bram.h

```diff
@@ -0,0 +1,57 @@
+// license:BSD-3-Clause
+// copyright-holders:Sergey Svishchev
+/***************************************************************************
+
+	Toshiba 1000 Backup RAM
+
+	Simulation of interface provided by 80C50 keyboard controller.
+
+***************************************************************************/
+
+#ifndef __TOSH1000_BRAM_H__
+#define __TOSH1000_BRAM_H__
+
+/***************************************************************************
+    DEVICE CONFIGURATION MACROS
+***************************************************************************/
+
+#define MCFG_TOSH1000_BRAM_ADD(_tag) \
+	MCFG_DEVICE_ADD(_tag, TOSH1000_BRAM, 0)
+
+
+#define BRAM_SIZE  160
```

Comment:
Please don't use `#define` if you can avoid it - there's no way to scope this stuff.